go/chat/convsource.go

```diff
@@ -140,12 +143,36 @@ func (s *HybridConversationSource) Pull(ctx context.Context, convID chat1.Conver
 					MsgID:          localData.Messages[0].GetMessageID(),
 				})
 				if err != nil {
-					return chat1.ThreadView{}, nil, err
+					return chat1.ThreadView{}, nil, nil, err
 				}
 				rl = append(rl, res.RateLimit)
 			}
 
-			return localData, rl, nil
+			var breaks []keybase1.TLFUserBreak
```

Comment:
Oh actually this is needed (i.e. scratch my comment above). Since we are not storing breaks in the levelDB. If this is called by GUI, previous pull could have broken proofs but still go through. Since the broken proofs are not (and probably shouldn't be) stored, there's no way here to know whether they are broken or not, which is why identify needs to be called again.